analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ExceptionShouldNotBeThrownFromUnexpectedMethods.CSharp11.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    interface IMyInterface : IDisposable
+    {
+        static event EventHandler OnSomeArrow
+        {
+            add => throw new Exception(); // Noncompliant
+            remove => throw new InvalidOperationException(); // Compliant - allowed exception
+        }
+    }
+
+    class MyClass : IMyInterface
```

Comment:
Why this test case, and in general why implementing IDisposable?

Reply:
I think it was left from the CSharp9 copied file,  my bad.